.gitignore

```diff
@@ -1,32 +1,18 @@
-lib-cov
-*.seed
-*.log
-*.csv
-*.dat
-*.out
-*.pid
-*.gz
-
-pids
-logs
-results
-
-npm-debug.log
-
+# Ignore generated folders
+.nyc_output/
+build/
+coverage/
+dist/
+node_modules/
+
+# Ignore compiled files
+components/**/*.bundle.js
+components/**/*.css
 public/css/
 public/js/*.js
 public/js/*.map
 
+# Ignore potential files
+.vscode/
```

Comment:
🎉

Reply:
We can also consider adding some `.vscode` configuration, for example recommended extensions and debug launch configurations.